test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenUpdateSpecs.java

```diff
@@ -82,7 +82,7 @@ public static void main(String... args) {
 	@Override
 	protected List<HapiApiSpec> getSpecsInSuite() {
```

Comment:
Do we need the changes in this file?

Reply:
these came from the `nft` branch. Can uncomment it tho.